<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CVCPelcoD</class>
 <widget class="QMainWindow" name="CVCPelcoD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CVC Visca over IP</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icon/icon/MainWindow.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>123</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1">
     <item row="0" column="1">
      <widget class="QLabel" name="lbPresetNo">
       <property name="text">
        <string>Preset</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lbCamNo">
       <property name="text">
        <string>Camera</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="camNo">
       <property name="font">
        <font>
         <pointsize>64</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="presetNo">
       <property name="font">
        <font>
         <pointsize>64</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="obsScene1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icon/icon/01D_CamOnly.png</normaloff>
      <normalon>:/icon/icon/01E_CamOnly.png</normalon>:/icon/icon/01D_CamOnly.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>39</width>
      <height>39</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="obsScene2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icon/icon/02D_TextBelow.png</normaloff>
      <normalon>:/icon/icon/02E_TextBelow.png</normalon>:/icon/icon/02D_TextBelow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>39</width>
      <height>39</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="obsScene3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icon/icon/03D_TextAbove.png</normaloff>
      <normalon>:/icon/icon/03E_TextAbove.png</normalon>:/icon/icon/03D_TextAbove.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>39</width>
      <height>39</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="obsScene4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icon/icon/04D_RightSlide1.png</normaloff>
      <normalon>:/icon/icon/04E_RightSlide1.png</normalon>:/icon/icon/04D_RightSlide1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>39</width>
      <height>39</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="obsScene5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icon/icon/05D_LeftSlide1.png</normaloff>
      <normalon>:/icon/icon/05E_LeftSlide1.png</normalon>:/icon/icon/05D_LeftSlide1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>39</width>
      <height>39</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="obsScene6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icon/icon/06D_RightSlide.png</normaloff>
      <normalon>:/icon/icon/06E_RightSlide.png</normalon>:/icon/icon/06D_RightSlide.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>39</width>
      <height>39</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="obsScene7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icon/icon/07D_LeftSlide.png</normaloff>
      <normalon>:/icon/icon/07E_LeftSlide.png</normalon>:/icon/icon/07D_LeftSlide.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>39</width>
      <height>39</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="obsScene8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icon/icon/08D_SlideOnCam.png</normaloff>
      <normalon>:/icon/icon/08E_SlideOnCam.png</normalon>:/icon/icon/08D_SlideOnCam.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>39</width>
      <height>39</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="obsScene9">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icon/icon/09D_SlideOnly.png</normaloff>
      <normalon>:/icon/icon/09E_SlideOnly.png</normalon>:/icon/icon/09D_SlideOnly.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>39</width>
      <height>39</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="obsScene10">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icon/icon/10D_Disable.png</normaloff>
      <normalon>:/icon/icon/10E_Disable.png</normalon>:/icon/icon/10D_Disable.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>39</width>
      <height>39</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="obsScene11">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icon/icon/11D_Begin.png</normaloff>
      <normalon>:/icon/icon/11E_Begin.png</normalon>:/icon/icon/11D_Begin.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>39</width>
      <height>39</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="sceneNo">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="iconSize">
     <size>
      <width>39</width>
      <height>39</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
